packages/rum/src/boot/recorderApi.spec.ts

```diff
@@ -162,6 +162,21 @@ describe('makeRecorderApi', () => {
       expect(stopRecordingSpy).toHaveBeenCalled()
     })
 
+    it('restarting the recording should keep using the same DeflateEncoder instance', () => {
+      setupBuilder.build()
+      rumInit()
+      recorderApi.start()
+
+      recorderApi.stop()
+      recorderApi.start()
+
+      // Types are specified to make sure we are getting the correct arguments
+      const firstCallDeflateEncoder: DeflateEncoder = startRecordingSpy.calls.argsFor(0)[4]
+      const secondCallDeflateEncoder: DeflateEncoder = startRecordingSpy.calls.argsFor(1)[4]
+
+      expect(firstCallDeflateEncoder).toBe(secondCallDeflateEncoder)
+    })
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏looking at this test alone, it is hard to understand why we want this behaviour.
Could we express a more high level behaviour?